frontend/src/main/pages/HomePage.js

```diff
@@ -25,6 +26,20 @@ export default function HomePage() {
         url: "/api/commons/all"
       }
     );
+  const onSuccess = (commons) => {
+    // Stryker disable next-line all : hard to get variable
+    let existed = new Boolean(false);
```

Comment:
Why are you doing `new Boolean` here? `false` should work just fine.

Reply:
changed